docs/platform/how-to/state-management.md

```diff
@@ -0,0 +1,70 @@
+# State management
+
+When deploying a service it is possible for that service to maintain state between restarts. This can be done by enabling state management for the deployment. Enabling state management creates a storage area that can be used for storage of data and files, and this state is preserved between service restarts.
+
+## Enabling state management
+
+When you deploy a service, it is possible to enable state management: 
```

Comment:
Enable state management during service deployment if your code relies on state or expects files to persist across restarts.

??